samples/oauth2-security/src/main/java/org/wso2/msf4j/example/Application.java

```diff
@@ -29,6 +29,9 @@
 
     private static final Logger logger = LoggerFactory.getLogger(Application.class);
 
+    private Application() {
+    }
+
```

Comment:
Thanks Vidura! Even though it would be technically correct, it might not be a convention to make the constructor of a Main class private. Please see below examples:

- https://github.com/wso2/carbon-kernel/blob/master/launcher/src/main/java/org/wso2/carbon/launcher/Main.java#L51
- https://github.com/ballerinalang/ballerina/blob/master/modules/launcher/src/main/java/org/ballerinalang/launcher/Main.java#L37
- https://github.com/spring-projects/spring-boot/blob/master/spring-boot-samples/spring-boot-sample-activemq/src/main/java/sample/activemq/SampleActiveMQApplication.java#L30
- https://github.com/jersey/jersey/blob/master/examples/sparklines/src/main/java/org/glassfish/jersey/examples/sparklines/Main.java#L61
- https://github.com/dropwizard/dropwizard/blob/master/dropwizard-benchmarks/src/main/java/io/dropwizard/benchmarks/util/DurationBenchmark.java#L18